04-Parallel-Simulations-With-MPI/01-Introduction-to-MPI.ipynb

```diff
@@ -0,0 +1,214 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to MPI\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "### Questions\n",
+    "\n",
+    "* What is MPI? \n",
+    "* Why should I run my simulations in parallel? \n",
+    "* How can I execute scripts in parallel?\n",
+    "\n",
+    "### Objectives\n",
+    "\n",
+    "* Describe **MPI**.\n",
+    "* Explain how **MPI** can provide faster **performance**, especially on **HPC** systems with **fast networks**.\n",
+    "* Show how to write a **single program** that can execute in serial or parallel.\n",
+    "* Demonstrate how to execute that program with **mpirun**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Introduction\n",
+    "\n",
+    "**MPI** (message passing interface) is a library that enables programs to execute in parallel in many CPUs and/or GPUs.\n",
+    "It is commonly available on **HPC** (high performance computing) clusters.\n",
+    "Using more resources in parallel can provide higher **performance** than the same simulation run on one CPU core or one GPU.\n",
+    "HOOMD-blue needs to communicate between the parallel processes that **MPI** executes, so a **fast network** like those on most **HPC** resources is necessary to obtain good performance."
```

Comment:
```suggestion
    "An important aspect of parallel simulations is to make sure all the processes that run in parallel get the input information they need. HOOMD-blue needs to communicate between the parallel processes that **MPI** executes, so a **fast network** like those on most **HPC** resources is necessary to obtain good performance."
```

I think it's necessary to explain a little bit why we care about the communication. For less experienced users, it may not be very obvious that it's a huge part of parallel simulations

Reply:
I think that mentioning communication here is too early. I've remove the reference to a "fast network".